src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -69,6 +69,12 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
                 psi.Arguments = testConsoleAppArgs;
             }
 
+            if (options.CollectConsoleOutput)
+            {
+                psi.RedirectStandardOutput = true;
+                psi.RedirectStandardError = true;
+            }
```

Comment:
I don't understand why this is needed.  Why not have the test just pass in the ProcessStartInfo configured the way it wants it?  That's what we do in other tests.

Reply:
Thanks! I just made the change to pass in ProcessStartInfo instead of doing this